lonestar/analytics/cpu/pagerank/PageRank-pull.cpp

```diff
@@ -258,27 +259,28 @@ void prResidual(Graph& graph) {
 
   initNodeDataResidual(graph, delta, residual);
   computeOutDeg(graph);
-  galois::StatTimer prTimer;
-  prTimer.start();
+
+  galois::StatTimer execTime("Timer_0");
```

Comment:
Just a comment, but this is one of those unfortunate cases where you have Timer_0 in more than one place due to how the code is laid out.